canals/pipeline/pipeline.py

```diff
@@ -569,15 +537,9 @@ def _run_component(self, name: str, inputs: Dict[str, Any]) -> Dict[str, Any]:
 
             input_class = instance.input
 
-            # If the node is variadic, unpack the input
-            if self.graph.nodes[name]["variadic_input"]:
-                inputs = list(inputs.values())[0]
-                input_dataclass = input_class(*inputs)
-
-            # Otherwise pass the inputs as kwargs after adding the component's own defaults to them
-            else:
-                inputs = {**instance.defaults, **inputs}
-                input_dataclass = input_class(**inputs)
+            # Pass the inputs as kwargs after adding the component's own defaults to them
+            inputs = {**instance.defaults, **inputs}
+            input_dataclass = input_class(**inputs)
```

Comment:
We removed the if statement, so we can just use `instance.input` and remove line 570